recipes/ubiquerg/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "ubiquerg" %}
+{% set version = "0.5.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "fb9f5cc40902d9327644471a7f5a69d2dcef36b3d4ea1485bc357ef4abdfcbaa"
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - python
+
+test:
+  imports:
+    - ubiquerg
+  requires:
```

Comment:
if you need ``veracitools`` to run this project, ``veracitools`` should be in ``run``

Reply:
is a package available here once the PR build is successful? Or it has to be merged first?

Reply:
It has to be merged, and it takes some time to get it build, packaged and uploaded to the anaconda cloud